evaluate_arima.py

```diff
@@ -0,0 +1,57 @@
+import warnings
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+from statsmodels.tsa.arima_model import ARIMA
+from sklearn.metrics import mean_squared_error
+
+from get_data.get_data import call_api
```

Comment:
👍 The linting here is correct, there should be a blank line between third party imports and application module imports :)